src/metrics.jl

```diff
@@ -208,46 +208,46 @@ kl_divergence(a::AbstractArray, b::AbstractArray) = evaluate(KLDivergence(), a,
 
 # RenyiDivergence
 function eval_start{T<:AbstractFloat}(::RenyiDivergence, a::AbstractArray{T}, b::AbstractArray{T})
-    zero(T), zero(T)
+    zero(T), zero(T), sum(a), sum(b)
```

Comment:
If `p` and `q` (`a` and `b`) should sum to 1, why do we need to compute their sum here?